frontend/css/listens-page.less

```diff
@@ -288,6 +287,11 @@
           border-radius: unset;
         }
       }
+      .add-cover-art {
+        .img {
+          max-height: 25px;
+        }
+      }
```

Comment:
I had a closer look after deploying the PR to test.listenbrainz.org

I found that we're not covering all cases with this css, and we end up with another cover art fallback image stretched like this:
<img width="311" alt="image" src="https://github.com/metabrainz/listenbrainz-server/assets/6179856/10a8c461-fc8b-40ec-bdd2-df18765a19f4">

So I dug into the css a little bit and found a more accurate solution that will cover all our fallback cases:
Instead of applying the 25px height to only the .add-cover-art image child, we need to apply `height: auto` to images in all three of the fallback classes.

The reason is somewhere else in the css definition meant for regular cover art, we set `height: inherit;` which is why we're seeing the issue. Resetting it with `auto` solves that issue without needing to add arbitrary pixel number like I initially suggested :)

it should go like this:
```less
      .add-cover-art,
      .not-mapped,
      .cover-art-fallback {
        [...]
        // We still want to remove "height: inherit;", keep that change
        > * {
          font-size: @cover-art-thumbnail-size / 2;
          max-width: @cover-art-thumbnail-size / 2;
          aspect-ratio: unset;
          border-radius: unset;
          height: auto; // Reset height here
        }
      }
```

And that covers all our cases!
<img width="474" alt="image" src="https://github.com/metabrainz/listenbrainz-server/assets/6179856/cb5a456a-95d3-4234-a3dd-efc28a851b4c">

Reply:
That was cool! Going ahead and patching this up

Reply:
I think you might have forgotten to push :)

Reply:
I did xD. Pushed!